### PR TITLE
[DOCS] Added shell autocomplete config instructions

### PR DESCRIPTION
**Pull request by jvanbuel**
How to configure autocomplete for the GE CLI

Changes proposed in this pull request:
-
-
-

After submitting your PR, CI checks will run and @tiny-tim-bot will check for your CLA signature.

For nontrivial changes, we will conduct both a design review and a code review. Please tag a core team member for code review, or leave blank and we will find someone for you!

- [ ] Design Review (@jcampbell)
- [x] Code Review

Thank you for submitting!

### CODE REVIEW

**Comment by jvanbuel** [comment]
If you guys want to keep it more concise, I can also remove the shell commands and just refer to the Click docs?

**Comment by kyleaton** [comment]
@cla-bot check

File: docs/command_line.rst [hunk 1]
```diff
@@ -957,6 +957,34 @@ If you have built a suite called ``churn_model_assumptions`` and a postgres data
 
 This tap can then be run nightly before your model makes churn predictions!
 
+Shell autocompletion for the CLI
+======================
```
Comment (Aylr) [inline docs/command_line.rst]: Nitpick: To make rst happiest the header needs to be at least as long as the text.
```suggestion
=================================
```

**Review by Aylr — APPROVED** [review]
This is such a boon to usability. Thanks a ton!

**Pull request merged**